graylog2-web-interface/src/components/common/ErrorAlert.tsx

```diff
@@ -33,8 +33,8 @@ const Container: StyledComponent<{margin: number}, ThemeInterface, HTMLElement>
 `);
 
 type Props = {
-  onClose: () => void,
-  children: JSX.Element | string,
+  onClose: (msg: string | undefined) => void,
```

Comment:
I find the following a bit more readable, not sure if we have any conventions on one or the other way:

```
  onClose: (msg?: string) => void,
```

Reply:
In this case it makes totally sense what you propose!